templates/webapps/galaxy/admin/index.mako

```diff
@@ -46,13 +46,30 @@
     <div class="unified-panel-body">
         <div class="toolMenu">
             <div class="toolSectionList">
+                    <div class="toolSectionTitle">Repositories</div>
+                    <div class="toolSectionBody">
+                        <div class="toolSectionBg">                        
+                        %if trans.app.tool_shed_registry and trans.app.tool_shed_registry.tool_sheds:
+                            <div class="toolTitle"><a href="${h.url_for( controller='admin_toolshed', action='browse_tool_sheds' )}" target="galaxy_main">Search Tool Shed</a></div>
```

Comment:
I'd group the three tool related entries under sever with what is currently repositories and name it "Tools and ToolShed". 

And, I think Server should be first.